app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
```

Comment:
Use correct return type. If your method raises an exception it doesn't mean it will return None